src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.kt

```diff
@@ -1,62 +1,70 @@
 package com.derongan.minecraft.deeperworld
 
+import com.derongan.minecraft.deeperworld.MinecraftConstants.FULL_DAY_TIME
 import com.derongan.minecraft.deeperworld.config.DeeperConfig
 import com.derongan.minecraft.deeperworld.services.WorldManager
 import com.derongan.minecraft.deeperworld.services.canMoveSections
 import com.mineinabyss.idofront.commands.CommandHolder
+import com.mineinabyss.idofront.commands.arguments.booleanArg
 import com.mineinabyss.idofront.commands.arguments.intArg
 import com.mineinabyss.idofront.commands.execution.ExperimentalCommandDSL
 import com.mineinabyss.idofront.commands.execution.IdofrontCommandExecutor
+import com.mineinabyss.idofront.commands.execution.stopCommand
 import com.mineinabyss.idofront.commands.extensions.actions.playerAction
 import com.mineinabyss.idofront.messaging.info
 import com.mineinabyss.idofront.messaging.success
 
 @ExperimentalCommandDSL
 object DeeperCommandExecutor : IdofrontCommandExecutor() {
     override val commands: CommandHolder = commands(deeperWorld) {
-        "sectionoff" {
-            playerAction {
-                player.canMoveSections = false
-                sender.success("Automatic TP disabled for ${player.name}")
-            }
-        }
-        "sectionon" {
-            playerAction {
-                player.canMoveSections = true
-                sender.success("Automatic TP enabled for ${player.name}")
+        ("deeperworld" / "dw") {
+            "tp"(desc = "Enables or disables automatic teleports between sections for a player") {
+                val state by booleanArg()
+                playerAction {
+                    if (state) {
+                        player.canMoveSections = true
+                        sender.success("Automatic TP enabled for ${player.name}")
+                    } else {
+                        player.canMoveSections = false
+                        sender.success("Automatic TP disabled for ${player.name}")
+                    }
+                }
             }
-        }
-        "linfo" {
-            playerAction {
-                val section = WorldManager.getSectionFor(player.location)
-                if (section == null)
-                    sender.info("${player.name} is not in a managed section")
-                else
-                    sender.info("${player.name} is in section ${section.key}")
+            "linfo"(desc = "Gets information about a players section") {
```

Comment:
Add an alias for it, `("layerinfo" / "linfo")`.

I'm also little concerned about everyone that already knows about the linfo command. Maybe it's worth adding some basic autocomplete so people can find it more easily, might honestly be worth keeping an error message for a little bit that says `"Please use /dw linfo or /deeperworld layerinfo instead"`.

Reply:
Yeah I agree. I'm also thinking of prioritizing autocompletion for now, its kind of essential for command use in the long run.

Reply:
For now literally just send back a list of all the options when the command name is deeperworld